### DISCUSSION TITLE
Clean Install on corrupted drive with unknown bios password?

### DISCUSSION THREAD

**Discussion by t-medz** [AveYo/MediaCreationTool.bat#251] [Q&A]
I have a laptop that I was about to send into Lenovo and see what they could do with the warranty to fix. 

The drive was first encrypted using full disk encryption with veracrypt. Then I botched the veracrypt rescue media and didn't set things up correctly to have the veracrypt key used to read the disk (or some such nonsense) -- point is it was inaccessible, and then I was trying to re-install windows so I went into diskpart and just wrecked that shit with merges and formats and finally a diskclean.. So now it just boots to the boot loader screen with no options. I can't access the bios because I set a password and of course forgot it. I can't get it to boot to any kind of external media. Would this allow me to re-install Windows 11? 

The device is a Lenovo Yoga 6 13ALC7

**Comment by rgiuffre** [comment]
Have you tried removing the BIOS password by doing a hard-reset on the motherboard? Since this is a laptop, just removing the main battery will not reset the BIOS password. You will need to disassemble the laptop and disconnect the coin cell battery on the motherboard. That may reset the BIOS password. Or check for Lenovo utilities to reset the BIOS password. You'll probably need to make a bootable USB drive to do so. Also, the Lenovo laptop may have a Fn key to hit during the boot process to let you choose which device to boot from. Usual suspects are F1, F2, F8, F12, or the Delete key.

**Reply by t-medz** [reply]
Yes, from my understanding there is no CMOS battery on this particular laptop. I did open it up to check then confirmed on other forums. When I called Lenovo support they had me press a small button on the bottom to go to a page I forgot the name but was basically a roundabout way to the BIOS so it still required the password. After this they told me I would need to send it in to use the warranty and possibly have the drive or mothervoard replaced. I've never heard of the Fn key to change the boot process though. Hopefully that would allow me to run a recovery drive. I have bootable usbs of Macrium and AOMEI which I'm hoping will allow me to rebuild the MBR and install a new OS.

**Reply by rgiuffre** [reply]
I found this: 
https://www.4winkey.com/computer-help/lenovo-boot-menu-key-on-windows-8-8-1-10.html
